src/libponyc/pass/expr.c

```diff
@@ -377,6 +377,7 @@ ast_t* find_antecedent_type(pass_opt_t* opt, ast_t* ast, bool* is_recovered)
     case TK_FUN:
     {
       ast_t* body = ast_childidx(parent, 6);
+      (void)body;
       pony_assert(ast == body);
```

Comment:
This feels like it should be wrapped in an ifdef so we dont do this extra lookup BUT... eh, does it matter as it is in the compiler and the ifdef would end up being weird.